protractor/helpers/reporter.ts

```diff
@@ -0,0 +1,18 @@
+import { SpecReporter, StacktraceOption } from 'jasmine-spec-reporter';
+const { AwesomeReport } = require('jasmine-awesome-report');
```

Comment:
use one of the two standards, either import or const

Reply:
I don´t have much experiencie with ts, i just configure with this https://github.com/aperdomob/jasmine-awesome-report

What is the correct way?

Reply:
Both are correct :) but when working, you should only use one. I recommend the import, because all of the other files have it

Reply:
import { AwesomeReport } from 'jasmine-awesome-report'); , is correct?